src/Components/Mobile/MobileMetricsListItem/MobileMetricsListItem.jsx

```diff
@@ -9,7 +9,7 @@ import Modal from "retail-ui/components/Modal";
 import type { Metric } from "../../../Domain/Metric";
 import MobileStatusIndicator from "../MobileStatusIndicator/MobileStatusIndicator";
 import roundValue from "../../../helpers/roundValue";
-import { Maintenances, type Maintenance } from "../../../Domain/Maintenance";
+import { Maintenances, type Maintenance, MaintenanceCaptions } from "../../../Domain/Maintenance";
```

Comment:
`MaintenanceTimes` and `MaintenanceCaptions` is private constant. They should not be exported. My mistake.

Let's remove "export" keyword in `Maintenance.js`. And replace  `MaintenanceCaptions` import by `getMaintenanceCaption`. And use this function in line 118